readme.md

```diff
@@ -215,6 +215,12 @@ Source control is not a secure place to store secrets such as credentials, API k
 - [Repo-supervisor](https://github.com/auth0/repo-supervisor) - _Auth0_ - Secrets scanning tool that can run as a CLI, as a Docker container or in AWS Lambda.
 - [truffleHog](https://github.com/trufflesecurity/truffleHog) - _Truffle Security_ - Searches through git repositories for secrets, digging deep into commit history and branches.
 
+### Supply Chain Security
```

Comment:
Not sure if @JakobTheDev will want this one moved, but the sections seem alphabetized so my assumption is that this should go under `Static Analysis`?